src/main/groovy/org/labkey/gradle/plugin/extension/TomcatExtension.groovy

```diff
@@ -49,36 +48,6 @@ class TomcatExtension
         return tomcatConfDir
     }
 
-    void validateCatalinaHome()
```

Comment:
@labkey-tchad Are the remaining references to the properties in `TomcatExtension` that are designated as used by TeamCity still relevant?

Reply:
They are not. I have a PR into this branch to remove them all: #217